lib/po.js

```diff
@@ -234,7 +236,24 @@ PO.parse = function (data) {
     return po;
 };
 
-PO.Item = function () {
+PO.parsePluralForms = function (pluralFormsString) {
+    var results = (pluralFormsString || '')
+        .split(';')
+        .reduce(function (acc, keyValueString) {
+            var trimmedString = keyValueString.trim();
+            var equalsIndex = trimmedString.indexOf('=');
+            var key = trimmedString.substring(0, equalsIndex).trim();
+            var value = trimmedString.substring(equalsIndex + 1).trim();
+            acc[key] = value;
+            return acc;
+        }, {});
+    return {
+        nplurals: results.nplurals,
+        plural: results.plural
+    };
+};
+
+PO.Item = function (nplurals) {
```

Comment:
Object for readability makes sense to me, and makes the parameter seem more optional.

I figured no one would be `new`-ing `PO.Item` directly, but if you'd prefer to keep the signature clean, I'm fine with doing an assignment after instantiation. In that case, perhaps I should use a setter (i.e., `Object.defineProperty`) to do the fallback to 2 logic?